### DISCUSSION TITLE
Sites as subdomains

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#3281] [General] [answered]
Hi, I am hosting Oqtane on a Windows web server with Plesk panel. When I create a new site in Oqtane using subdomain, how do I address it from Plesk? Has anyone had that experience?

For example, the main domain is called www.maindomain.com, then I have created a subdomain called client1.maindomain.com in Plesk, but Plesk gives me 3 options: 1. Website, 2. Redirect, 3. No website.

How do I get that subdomain directed to the correct Oqtane site.

The same applies if I wanted to do it directly in IIS, how could I achieve this?

Thanks

**Comment by W6HBR** [comment]
In IIS you would add another binding on the same port (same site), but with a different host name.  Your DNS recs can be either an A or a CNAME that either point to the same IP (A rec) or point to the primary host (CNAME) of www.maindomain.com.  IIS lets you use multiple DNS names for the same site.  Oqtane will take it from there.  You can have multiple names for the same site in Oqtane or it will direct it to the correct named subdomain (site).

I can't tell you specifically for Plesk (as I don't use it), but you'd want to be similar to how IIS treats it natively.  e.g. just point your DNS to the same host.  If one of the Plesk options allows that, then you're set.

**Reply by LuisVilaplana** [reply, marked as answer]
Yes it worked! Thank you very much. This is what I did:

1. In Plesk you do not have to create any subdomains, the only thing you have to do is create a CNAME in the DNS records like this:

Type: CNAME
Domain: alianza
Canonical name: prosoftpos.net

2. In IIS, you only have to create one binding for each subdomain you create in Oqtane. In my case I only made a binding on port 443 for each subdomain and assigned the certificate of the main domain

![image](https://github.com/oqtane/oqtane.framework/assets/103559699/f28daffe-21f2-4ee7-9763-0d0019ea1641)

3. In Oqtane, you must create a site for each subdomain, like this:

Name: The short name of your site. ie: Alianza
Alias: The address of the site. ie:  alianza.prosoftpos.net

The rest is done according to taste or need. In my case I created a separate SQL Server database for each subdomain

That's all. Don't rush, there are propagation times for DNS records, so it won't always work immediately.

Thank you all, this also resolved the discussion #3265 regarding mixing sessions on different tenants, at least on production server.
